torch/onnx/__init__.py

```diff
@@ -314,6 +314,10 @@ def forward(self, x):
             If True,  argument ``f`` must be a string specifying the location of the model.
             The external data files will be stored in the same directory as ``f``.
             This argument is ignored unless ``operator_export_type=OperatorExportTypes.ONNX``.
+
+    Raises:
+      ONNXCheckerError: If the ONNX checker detects an invalid ONNX graph. And the proto file will
+        still be generated even this error occurred.
```

Comment:
`If the ONNX checker detects an invalid ONNX graph. Will still export the model to the file ``f`` even if this is raised.`